### PR TITLE
Improve math_ops.py docs

### PR DESCRIPTION
**Pull request by stmugisha**

### CODE REVIEW

**Comment by steph-en-m** [comment]
@yashk2810 can you help re-review this. I've cleared some commits off the log from the previous pull request (referenced above). Thanks

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -3288,32 +3305,50 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
...
+  array([[ 2,  6, 12, 20],
+         [ 1,  4,  9, 16]], dtype=int32)>
+ 
   By setting the `exclusive` kwarg to `True`, an exclusive cumsum is performed
   instead:
-
-  ```python
-  tf.cumsum([a, b, c], exclusive=True)  # [0, a, a + b]
-  ```
+  # tf.cumsum([a, b, c], exclusive=True)  => [0, a, a + b]
```
Comment (yashk2810) [inline tensorflow/python/ops/math_ops.py]: Can you please add a newline before this comment and everywhere else?

Something like this
```
By setting the `exclusive` kwarg to `True`, an exclusive cumsum is performed instead:

# tf.cumsum([a, b, c], exclusive=True)  => [0, a, a + b]
```

Comment (steph-en-m) [reply tensorflow/python/ops/math_ops.py]: changes implemented as requested

**Review by yashk2810 — DISMISSED** [review]

File: tensorflow/python/ops/math_ops.py (hunk 1 above)
Comment (steph-en-m) [reply tensorflow/python/ops/math_ops.py]: The Ubuntu CPU test was failing at tf.cumsum() so i made minor tweaks to make it pass.

**Comment by steph-en-m** [comment]
The ubuntu cpu test was failing at tf.cumsum so I've made minor changes to the doctest to make it pass.

**Review by yashk2810 — DISMISSED** [review]

**Comment by steph-en-m** [comment]
ubuntu sanity check was failing due to a pylint error(`long line 82/80`) and that has been fixed in the commit above. Is it by design that the `tf_doctest.py` doesn't capture these?

**Review by rthadur — APPROVED** [review]

**Pull request merged**